shell/client/shell.html

```diff
@@ -41,11 +41,41 @@
   </div>
   {{else}}
   <div id="topbar-right">
+    {{#if currentUser}}
+      {{> notifications}}
+    {{/if}}
     {{> loginButtons align="right"}}
   </div>
   {{/if}}
 </template>
 
+<template name="notifications">
+  <span class="notification-topbar {{#if notificationsIsEnabled}}notification-selected{{/if}}">
+    <button class="notification-button {{#if notificationCount}}inner{{else}}notification-icon{{/if}}">{{notificationCount}}</button>
+  </span>
+  {{#if notificationsIsEnabled}}
+    <div id="notification-dropdown">
+      <h4>Notifications</h4>
+      <ul id="notification-list">
+        {{#each notifications}}
+          <li>
+            <div class="notification-item">
+              <div class="notification-header">
+                  <span>{{grainTitle}}</span>
+                  <span title="Dismiss" class="cancel-notification" data-notificationid="{{_id}}">x</span>
```

Comment:
We'll need to make it clear that the user is canceling the task, not simply dismissing the notification. Designing this is probably a task for Nena but for the moment maybe we should at least use a button labeled "cancel" rather than a little red x.